src/lib/api/event.replayer.ts

```diff
@@ -189,8 +189,18 @@ export class EventReplayer {
     prebootData.apps = [];
     this.clientNodeCache = {};
 
-    // sent event to documernt that signals preboot complete
-    const completeEvent = new Event('PrebootComplete');
+    // send event to document that signals preboot complete
+    const completeEventName = 'PrebootComplete';
+    let completeEvent: CustomEvent;
+    // constructor is not supported by older browsers ( i.e. IE9-11 )
+    // in those browsers, the type of CustomEvent will be "object"
+    if (typeof CustomEvent === 'function') {
```

Comment:
Consolidate all of this logic into the `if` statement as follows. Also, we're not going to provide a half-baked IE option when a polyfill is readily available.

```typescript
// constructor is not supported by older browsers ( i.e. IE9-11 )
// in those browsers, the type of CustomEvent will be "object"
if (typeof CustomEvent === 'function') {
  // send event to document that signals preboot complete
  const completeEvent = new CustomEvent('PrebootComplete');
  doc.dispatchEvent(completeEvent);
}

Reply:
To clarify, you want to remove the fallback option? If so, should the need of a polyfill be documented?

Reply:
Yes, please add a section to the README and remove the fallback option. In either case (using the fallback or not), this would've needed to be documented because it's behavior that deviates from expectation.

Reply:
Ok, no problem. I will make the changes ASAP.